reference/promise-types/files.markdown

```diff
@@ -7,7 +7,78 @@ tags: [reference, bundle agent, files, promises, files promises, promise types]
 
 Files promises manage all aspects of files. Presence, absence, file content, permissions, and ownership. File content can be fully or partially managed.
 
-[%CFEngine_include_example(files_content.cf)%]
+## Examples
+
+### Creating a file with content
+
+```cf3
+bundle agent __main__
+{
+  files:
+    "/tmp/hello"
+      content => "Hello, CFEngine";
+}
+```
+
+Before making changes to the system, CFEngine will check what the current state is, and whether it matches the desired state.
+In this case, CFEngine will create and edit the file only if necessary, if `/tmp/hello` already exists with the correct content, no changes are made.
+
+**Tip:** In this and other examples, we use the `__main__` bundle.
+This allows you to easily run a policy file directly.
```

Comment:
```suggestion
This bundle is evaluated when (and only when) running the policy file directly.
```

Reply:
No, my example is easier / more focused on the user, IMO.